### PR TITLE
OCPBUGS-39249: fix Sort all resources list

### PR DESCRIPTION
**Pull request by upalatucci**
Sorting method to sort matchLabels pod selector in network policies

specify an ingress method for hosts

fix small things for routes and service

### CODE REVIEW

File: src/utils/utils/sorting.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import { K8sResourceCommon } from '@openshift-console/dynamic-plugin-sdk';
+
+const getValueByPath = (obj: K8sResourceCommon, path: string) => {
+  const pathArray = path?.split('.');
+  return pathArray?.reduce((acc, field) => acc?.[field], obj);
+};
+
+export const sorting = (path: string, direction: string) => (a, b) => {
```
Comment (pcbailey) [inline src/utils/utils/sorting.ts:8]: This needs a more descriptive name and it should be phrased as a verb. `sortIngressesByHost` below is a great example.

Comment (upalatucci) [reply src/utils/utils/sorting.ts]: is `sortingCommonColumnsByPath` okay?

File: src/utils/utils/sorting.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import { K8sResourceCommon } from '@openshift-console/dynamic-plugin-sdk';
+
+const getValueByPath = (obj: K8sResourceCommon, path: string) => {
+  const pathArray = path?.split('.');
+  return pathArray?.reduce((acc, field) => acc?.[field], obj);
+};
+
+export const sortingCommonColumnsByPath = (path: string, direction: string) => (a, b) => {
```
Comment (pcbailey) [inline src/utils/utils/sorting.ts:8]: In this case it would be `sortCommonColumnsByPath`.

**Comment by metalice** [comment]
fine by me , just rename Phillip suggestion please

**Comment by upalatucci** [comment]
renamed @pcbailey @metalice

**Review by metalice — APPROVED** [review]

**Pull request merged**